_data/overviews.yml

```diff
@@ -47,11 +47,16 @@
       url: "core/architecture-of-scala-213-collections.html"
       by: Julien Richard-Foy
       description: "These pages describe the architecture of the collections framework introduced in Scala 2.13. Compared to the Collections API you will find out more about the internal workings of the framework."
-    - title: Custom Collection Types
+    - title: Implementing a Custom Collection (Scala 2.13)
       icon: building
       url: "core/custom-collections.html"
       by: Martin Odersky, Lex Spoon and Julien Richard-Foy
       description: "In this document you will learn how the collections framework helps you define your own collections in a few lines of code, while reusing the overwhelming part of collection functionality from the framework."
+    - title: Generic Programming With Collections (Scala 2.13)
```

Comment:
I think we should find a different title. The term "generic programming" in the context of Scala is claimed by shapeless. Reuising it can cause confusion.

Something like "Writing generic, external operations that integrate with all collection types"

Reply:
Thanks, I’ve changed the title for “Programming with collections generically”

Reply:
still not a good title IMO, see my other comment on that